scripts/supabase-remote-init.mjs

```diff
@@ -57,11 +60,47 @@ async function createProject({ projectName, databasePassword }) {
         }
     );
 
-    const projectRef = stdout.match(
-        /https:\/\/supabase.com\/dashboard\/project\/([a-zA-Z0-9_-]*)/
-    )[1];
+    try {
+        const matchJSON = stdout.match(new RegExp('{.*}', 's'));
+        if (!matchJSON) {
+            throw new Error('Invalid JSON output');
+        }
+        const jsonOuput = JSON.parse(matchJSON[0]);
+        return jsonOuput.id;
+    } catch (e) {
+        console.error('Failed to create project');
+        console.error(e);
+        throw e;
+    }
+}
 
-    return projectRef;
+async function waitForProjectToBeReady({ projectRef }) {
+    console.log('Waiting for project to be ready...');
```

Comment:
Not a debug log. The wait can be long so we display this message to let users know we're not stuck